tck/bridge-tck-common/src/main/java/com/liferay/faces/bridge/tck/common/util/BridgeTCKUtil.java

```diff
@@ -26,15 +26,13 @@
 public class BridgeTCKUtil {
 
 	public static boolean isHeaderOrRenderPhase(Bridge.PortletPhase portletPhase) {
-		return (Bridge.PortletPhase.HEADER_PHASE.equals(portletPhase) ||
-			Bridge.PortletPhase.RENDER_PHASE.equals(portletPhase));
+		return Bridge.PortletPhase.HEADER_PHASE.equals(portletPhase);
```

Comment:
@ngriffin7a, yes. I didn't change the method name because I thought maybe it would cause unnecessary diffs. But perhaps we could just inline the code and old versions would have `RENDER_PHASE` while the new version would have `HEADER_PHASE`.